RealmSwift/RealmCollection.swift

```diff
@@ -45,6 +45,39 @@ import Realm
     }
 }
 
+/// A container used for storing key-value pairs which are to be used as elements in `RLMMapIterator`.
```

Comment:
This appears to be something which should be `:nodoc:` as it's just an implementation detail that users shouldn't know about and would be internal if Swift let us.